Live-Snap/Live-Snap/Networking/SnapcodeEndpointRequest.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SnapcodeEndpointRequest.swift
+//  Live-Snap
+//
+//  Created by Oleg Abalonski on 1/6/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+
+class SnapcodeEndpointRequest {
+    
+    var snapchatUsername: String
+    
+    init(snapchatUsername: String) {
+        self.snapchatUsername = snapchatUsername
+    }
+    
+    //https://github.com/Alamofire/AlamofireImage
+    //Shouldn't we be Almofire.download the image???
+    // Make a network call to api and return a snapcode image if successful
+    func start(completion: @escaping ((UIImage?) -> ())) {
+        
+        let url = "https://feelinsonice-hrd.appspot.com/web/deeplink/snapcode?username=\(snapchatUsername)&type=PNG&size=800"
+    
+        // Make Alamofire request
+        Alamofire.request(url).responseData { response in
```

Comment:
for input arguments into a block I like to give the variables types so you know what they are, here it's unclear what the type of the response variable is. 

you can add the type like this 
Alamofire.request(url).responseData { (response: TypeOfResponse) in